fotoapparat/src/main/java/io/fotoapparat/routine/UpdateOrientationRoutine.java

```diff
@@ -41,7 +46,11 @@ public void onOrientationChanged(final int degrees) {
         cameraExecutor.execute(new Runnable() {
             @Override
             public void run() {
-                cameraDevice.setDisplayOrientation(degrees);
+                try {
+                    cameraDevice.setDisplayOrientation(degrees);
+                } catch (RuntimeException e) {
+                    cameraErrorCallback.onError(new CameraException(e));
```

Comment:
There is a problem with that - at the moment `onError` basically means "everything is super bad, camera is dead". However, it is very likely not the case here, since orientation events are constantly coming in.

I would instead do it like it is done in `Camera1#autoFocus` where we just log the failure in `logFailedAutoFocus`.

Reply:
Ok. I'll make that change.